kogito-editors-js/packages/import-java-classes-component/src/components/ImportJavaClasses/ImportJavaClassesWizardSecondStep.css

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.loader {
```

Comment:
In the new boxed expression editor, we generally try to use a namespace on all CSS classes, see: https://github.com/kiegroup/kogito-editors-java/blob/main/kogito-editors-js/packages/boxed-expression-component/src/components/FunctionExpression/EditParameters.css#L17.

This is quite important, otherwise, when we integrate this component on the DMN editor, the CSS classes may leak and may impact other components that have a `loader` class in the DMN editor.

Thus, for this component, we could have a `.import-java-classes` prefix and define this style like `.import-java-classes .loader` :-)

Reply:
@karreiro Done, thank you